scapy/arch/common.py

```diff
@@ -68,32 +84,47 @@ def get_if_raw_hwaddr(iff, siocgifhwaddr=None):
     :returns: the corresponding raw MAC address
     """
     if siocgifhwaddr is None:
-        from scapy.arch import SIOCGIFHWADDR
+        from scapy.arch import SIOCGIFHWADDR  # type: ignore
```

Comment:
We should try to never import platform-specific constants from `scapy.arch` but rather use `scapy.arch.linux....` (etc.) instead. This is an exception because this changes depending on the platform, but can't be added to `__all__` as it's not available on platforms like Windows.

Reply:
Actually we would also have to add it to BSD and I don't think it's worth it to have 2 dummys floating around.